fbgemm_gpu/test/quantize_ops_test.py

```diff
@@ -871,5 +872,39 @@ def test_quantize_and_dequantize_op_cuda_large_nrows_bf16(
             torch.testing.assert_allclose(dequantized_data_gpu.cpu(), dequantized_data)
 
 
+class TestDenseMLPQuantizationConversion(unittest.TestCase):
```

Comment:
Remove this to sync with the trunk?

Reply:
When enabling the test, an error is thrown `RuntimeError: No such operator fb::FloatToMSFPQuantized`. Is there any specific version of Pytorch we need to test?

Reply:
We need to use torch.ops.fbgemm.FloatToMSFPQuantized